website/client/src/components/settings/site.vue

```diff
@@ -739,6 +739,14 @@ export default {
       } else if (attribute === 'email') {
         this.user.auth.local.email = updates.newEmail;
         window.alert(this.$t('emailSuccess')); // eslint-disable-line no-alert
+      } else if (attribute === 'password') {
+        this.passwordUpdates = {};
+        this.$store.dispatch('snackbars:add', {
+          title: 'Habitica',
+          text: "Your password has been changed",
```

Comment:
```suggestion
          text: 'Your password has been changed',
```
change to single quote